components/Faq.js

```diff
@@ -1,7 +1,134 @@
 import React from "react";
+import Accordion from "react-bootstrap/Accordion";
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "../tailwind.config.js";
+
+const fullConfig = resolveConfig(tailwindConfig);
```

Comment:
what is this for?

Reply:
I originally had it for the background style for the last accordion element, but it seems to be working fine w/o bgStyle, so I've just taken it out.